home/screens/BranchDetailsScreen/BranchDetailsView.tsx

```diff
@@ -0,0 +1,79 @@
+import { spacing } from '@expo/styleguide-native';
+import { StackScreenProps } from '@react-navigation/stack';
+import dedent from 'dedent';
+import { Divider, Text, useExpoTheme, View } from 'expo-dev-client-components';
+import * as React from 'react';
+import { ActivityIndicator } from 'react-native';
+
+import ScrollView from '../../components/NavigationScrollView';
+import { RedesignedSectionHeader } from '../../components/RedesignedSectionHeader';
+import { UpdateListItem } from '../../components/UpdateListItem';
+import { BranchDetailsQuery } from '../../graphql/types';
+import { HomeStackRoutes } from '../../navigation/Navigation.types';
+import { BranchHeader } from './BranchHeader';
+import { EmptySection } from './EmptySection';
+
+const ERROR_TEXT = dedent`
+  An unexpected error has occurred.
+  Sorry about this. We will resolve the issue as soon as possible.
+`;
+
+type Props = {
+  loading: boolean;
+  error?: Error;
+  data?: BranchDetailsQuery;
+} & StackScreenProps<HomeStackRoutes, 'BranchDetails'>;
+
+export function BranchDetailsView({ loading, error, data }: Props) {
```

Comment:
should this be a flat list or is it mostly a small amount of items rendered in the scroll view

Reply:
I think you're right that this should be a flatlist! I'll update this